### PR TITLE
Get the project passing jshint

### PR DESCRIPTION
**Pull request by danvk**
Lots of small style tweaks here. Consider these proposed rules: this is a good opportunity to voice your objection to any you don't like!

### CODE REVIEW

File: viewer/d3.stacked-bar-chart.js [hunk 1]
```diff
@@ -1,276 +1,282 @@
-(function(){
+(function() {
 
-function sum(arr) {
+  function sum(arr) {
```
Comment (ihodes) [inline viewer/d3.stacked-bar-chart.js:3]: I'm a bit opposed to indenting all the code inside the closure/pseudomodule `(function(){})()` wrapper--makes it just a little harder to read, extends the lines by 2 unnecessary characters, all for no benefit (in my eyes).

**Comment by ihodes** [comment]
Looks good (other than my one issue with jshint—what's your opinion?)

File: viewer/d3.stacked-bar-chart.js (hunk 1 above)
Comment (danvk) [inline viewer/d3.stacked-bar-chart.js:3]: Yeah, I agree. I've filed a feature request against JSCS to add something for this: https://github.com/mdevils/node-jscs/issues/436

**Comment by danvk** [comment]
As it turns out, jshint is moving away from being a way to check coding style. (See https://github.com/jshint/jshint/issues/1358). JSCS (JavaScript Code Style) is a tool that's specifically built to enforce style guides. It has a few presets (like Google's). I changed this pull request to use that, in addition to jshint.

**Comment by ihodes** [comment]
Looks good, pending the inside-closure spaces. Sounds like JSCS is a win for our js dev, pending that issue.

**Comment by danvk** [comment]
I'll merge while we wait to hear from the JSCS folks.

**Pull request merged**